.github/workflows/previews-build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: install dependencies
-        uses: "gradio-app/gradio/.github/actions/install-all-deps@main"
+        uses: "gradio-app/gradio/.github/actions/install-all-deps"
```

Comment:
why did you remove @main?

Reply:
I think untagged and tagged with main are equivalent but it is better to be explicit.

Reply:
Oh I removed it for testing this branch and it has to be reverted.